librina/src/netlink-messages.cc

```diff
@@ -137,16 +137,25 @@ void BaseNetlinkMessage::setResponseMessage(bool responseMessage) {
 
 const std::string BaseNetlinkMessage::toString(){
 	std::stringstream ss;
-	ss << "Family: " << family << "; Operation code: "
+	std::string type;
+	if (requestMessage) {
+		type = "Request";
+	} else if (responseMessage) {
+		type = "Response";
+	} else if (notificationMessage) {
+		type = "Notification";
+	} else {
+		type = "Unknown";
+	}
+
+	ss << "Fam: " << family << "; Opcode: "
 			<< BaseNetlinkMessage::operationCodeToString(operationCode)
-			<< "; Source port: " << sourcePortId
-			<< "; Destination port: " << destPortId
-			<< "; Sequence Number: " << sequenceNumber << "\n"
-			<< "Is request message? " << requestMessage
-			<< "; Is response message? " << responseMessage
-			<< "; Is notification message? " << notificationMessage << "\n"
-			<< "Source IPC Process: " << sourceIPCProcessId
-			<< "; Destination IPC Process: " << destIPCProcessId;
+			<< "; Sport: " << sourcePortId
+			<< "; Dport: " << destPortId
+			<< "; Seqnum: " << sequenceNumber
+			<< "; " << type
```

Comment:
No, I didn,t since it is already understood by the context (and the intend was to make the log as compact as possible)